tests/run_astyle.sh

```diff
@@ -1,4 +1,17 @@
 #!/bin/bash
 # SPDX-License-Identifier: MIT
 
-find src tests -name '*.[ch]' | grep -v '/external/' | grep -v 'kem/.*/.*/.*' | grep -v 'sig/.*/.*/.*' | xargs astyle --dry-run --options=.astylerc
+# check style of non-external code:
+find src tests -name '*.[ch]' | grep -v '/external/' | grep -v 'kem/.*/.*/.*' | grep -v 'sig/.*/.*/.*' | xargs astyle --dry-run --options=.astylerc | grep Format
+if [ $? -ne 1 ]; then
+   echo "Error: Some files need reformatting. Check output above. Exiting."
+   exit -1
```

Comment:
Suggest not exiting here but saving return value and outputting it at the end, so that the user also gets to see the results of running the next check on CRLF.

Reply:
Makes sense. See new commit.